src/test/java/com/openlattice/serializer/AbstractJacksonYamlSerializationTest.java

```diff
@@ -44,13 +47,12 @@ public void testSerdes() throws IOException {
         T data = getSampleData();
         YamlSerializationResult<T> result = serialize( data );
         logResult( result );
-        Assert.assertEquals( data, result.deserializeJsonString( getClazz() ) );
-        Assert.assertEquals( data, result.deserializeJsonBytes( getClazz() ) );
-        Assert.assertEquals( data, result.deserializeSmileBytes( getClazz() ) );
-        Assert.assertEquals( data, result.deserializeYamlString( getClazz() ) );
```

Comment:
What is the problem with `assertEquals` ?

Reply:
There's no problem with `assertEquals` in general but I wanted to expose a way for callers to override how equals is performed. 
Specifically this disgusting nonsense I had to do:
https://github.com/openlattice/linker/pull/41/files#diff-bb44e7b36825425a50e58fc051e76984R63

Reply:
😢Ohhh, I see. Thanks for the explanation!